package.json

```diff
@@ -60,9 +61,9 @@
     "@zeit/next-css": "^1.0.1",
     "apollo-server": "^2.9.5",
     "apollo-server-express": "^2.9.5",
-    "express": "^4.17.1",
```

Comment:
Oh no! I don't know how this got deleted, but the `express` package is still in use so we don't need to remove this.